src/com/oltpbenchmark/DBWorkload.java

```diff
@@ -899,17 +898,43 @@ private static Results runWorkload(List<BenchmarkModule> benchList, boolean verb
             }
         }
         LOG.info(SINGLE_LINE);
-        LOG.info("Rate limited reqs/s: " + r);
 
-        long tpmc = numNewOrderTransactions * 60 / time;
+        double tpmc = 1.0 * numNewOrderTransactions * 60 / time;
         double efficiency = 1.0 * tpmc * 100 / numWarehouses / 12.86;
 
-        LOG.info("Num New Order transactions : " + numNewOrderTransactions + ", time seconds: " + time);
+        LOG.info("Throughput: " + r + " Num New Order transactions : " + numNewOrderTransactions + ", time seconds: " + time);
         LOG.info("TPM-C: " + tpmc);
```

Comment:
Can you use just 2 digits after decimal point while printing so that it looks neater and more readable?